scripts/resolve-modules.sh

```diff
@@ -2,7 +2,8 @@
 # Recursively finds all directories with a go.mod file and creates
 # a GitHub Actions JSON output option. This is used by the linter action.
 
+# Based on https://github.com/golangci/golangci-lint/issues/828#issuecomment-658207652
 echo "Resolving modules in $(pwd)"
 
-PATHS=$(find . -mindepth 2 -type f -name go.mod -printf '{"workdir":"%h"},')
+PATHS=$(find . -mindepth 2 -type f -name go.mod -printf '{"workdir":"%h"},' | sed 's/\.\///g')
```

Comment:
This removes the `./` prefix from the directories, makes it easier to use later on in the GitHub action

Reply:
Nice 👍